README.md

```diff
@@ -10,47 +23,30 @@ It depends.
 
 The plugin, as it is currently implemented is meant to be a very simple plugin to demonstrate how to get started with GoCD [elastic agent](https://plugin-api.go.cd/current/elastic-agents) feature. This plugin terminates docker containers very aggressively (within a minute or two of the agent being idle). Depending on your usage, this may not be desirable. If this behavior is undesirable to you, you may need to fork this plugin and [tweak it a bit](https://github.com/gocd-contrib/docker-elastic-agents/blob/master/src/main/java/cd/go/contrib/elasticagents/docker/executors/ServerPingRequestExecutor.java) so the docker containers are not terminated as aggressively.
 
-## Customizing your docker image to run as a GoCD Elastic Agent
-
-There are two ways to customize your docker image to work with this plugin
- 
-### Use the GoCD agent
+## Using your own docker image with elastic agents
 
-* Ensure that you have installed the go-agent for your distribution, using apt/yum or a zip file if you're using a distribution that does not support apt or yum.
-* Once the agent is installed, create a simple shell script executed via (`CMD`) that will accept the following variables and execute the agent bootstrapper process. These environment variables are passed by this plugin when performing a `docker run`. Your image is expected to use these variables to create a correct `autoregister.properties`:
-  * `GO_EA_SERVER_URL` - the URL of the GoCD server (from the plugin settings page)
-  * `GO_EA_AUTO_REGISTER_KEY` - the auto-register key
-  * `GO_EA_AUTO_REGISTER_ENVIRONMENT` - the auto-register environment
-  * `GO_EA_AUTO_REGISTER_ELASTIC_AGENT_ID` - the elastic agent id
-  * `GO_EA_AUTO_REGISTER_ELASTIC_PLUGIN_ID` — the elastic plugin id
+The plugin executes the equivalent of the following docker command to start the agent —
 
-### Use the GoCD golang bootstrapper
+```
+docker run -e GO_EA_SERVER_URL=...
+           -e GO_EA_AUTO_REGISTER_KEY=...
+           -e GO_EA_AUTO_REGISTER_ENVIRONMENT=...
+           -e GO_EA_AUTO_REGISTER_ELASTIC_AGENT_ID=...
+           -e GO_EA_AUTO_REGISTER_ELASTIC_PLUGIN_ID=...
+           IMAGE_ID
+```
 
-This method is a [bit insecure (PR welcome)](https://github.com/ketan/gocd-golang-bootstrapper), but uses lesser memory and boots up and starts off a build quickly:
- 
-```dockerfile
-FROM yourimage
+Your docker image is expected to contain a bootstrap program (to be executed via docker's `CMD`) that will create an [`autoregister.properties`](https://docs.gocd.io/current/advanced_usage/agent_auto_register.html) file using these variables. The `GO_EA_SERVER_URL` will point to the server url that the agent must communicate with.
```

Comment:
Link to something like [this](https://github.com/ketan/docker-elastic-agents/blob/92db09abf6b313eaca6969e374f9ef37f9e9c269/contrib/scripts/bootstrap-via-installer/go-agent#L26) here. It'll help understand what you're saying, better.